### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jshilts657**
Readme updates
Updated tools and instructions on how to run the software

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +90,22 @@ For more information please refer to this link: https://docs.python.org/3/librar
...
+
+3. To run this application the services must be started in the correct order or an error may occur. The first service to start is Python Flask
+   a. `cd backend` or navigate to backend directory
+   b. To run the application using python use `python3 backend.py`
+   c. To run the application using Docker or Podman run the commands below
+      i. docker build -t flask_backend .
+      ii. docker run -t flask_backend
+   
+4. To Run the frontend application you need be in the /code/frontend directory.
+     a. To run the application using python use `npm start`
```
Comment (rzheng2019) [inline README.md]: This should just be Node, not Python, when running the frontend application.

File: README.md [hunk 2]
```diff
@@ -79,7 +90,22 @@ For more information please refer to this link: https://docs.python.org/3/librar
...
+     b. To run the application using Docker or Podman run the commands below
+      i. docker build -t front_end .
+      ii. docker run  -d -t front_end
```
Comment (rzheng2019) [inline README.md]: Looks good, although there looks like an extra space between "run" and "-d"

**Review by rzheng2019 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jshilts657) [reply README.md]: Changed to Node

File: README.md (hunk 2 above)
Comment (jshilts657) [reply README.md]: extra space removed
Also updated to use md format which  I forgot to do

**Review by rzheng2019 — APPROVED** [review]

**Review by BlueGreenDragon — APPROVED** [review]
Looks good.

**Review by Rekik27 — APPROVED** [review]
looks great

**Pull request merged**